data/summative-questionnaires/brain-development-and-how-children-learn.yml

```diff
@@ -30,7 +30,8 @@ brain-development-and-how-children-learn:
           
           After birth, when a baby or young child repeats and practises a skill, it leads to myelination of the neurones involved. This increases the speed at which the message travels between neurones and so improves the skill. 
 
-          You might want to revisit the section on brain development before birth.
+          You might want to revisit the section on [brain development before birth](/modules/brain-development-and-how-children-learn/content-pages/2-1-1){:class="govuk-link"}.
```

Comment:
@ebrett @agoldstone93 I think we should tweak the SCSS so links within the govspeak class get styled by default. What do you think?

Reply:
I think that might already be the case, possibly redundant to also have the class here see https://github.com/DFE-Digital/early-years-foundation-recovery/blob/main/app/assets/stylesheets/govspeak/helpers/_markdown-typography.scss

```
  a {
    @include govuk-link-style-default;

    &:focus {
      @include govuk-focused-text;
    }
  }
```

Reply:
By Jove!